src/templates/navbar-fixed-admin-snippet.html

```diff
@@ -7,9 +7,14 @@
         <li><a href="{% url 'courses:course_list' %}"><i class="fa fa-fw fa-book"></i>&nbsp;Courses</a></li>
         <li><a href="{% url 'courses:semester_list' %}"><i class="fa fa-fw fa-calendar-o"></i>&nbsp;Semesters</a></li>
         <li role="separator" class="divider"></li>
+        <li class="dropdown-header">Custom Pages</li>
+        <li><a href="{% url 'utilities:flatpage-list' %}"><i class="fa fa-list"></i>&nbsp;Pages</a></li>
+        <li><a href="{% url 'utilities:flatpage-create' %}"><i class="fa fa-pencil-square-o"></i>&nbsp;Create Page</a></li>
+        <li role="separator" class="divider"></li>
         <li class="dropdown-header">Advanced</li>
         <li><a href="/admin"><i class="fa fa-fw fa-database"></i>&nbsp;Site Administration</a></li>
         <li role="separator" class="divider"></li>
         <li class="dropdown-header">ByteDeck</li>
-        <li><a title="Ask a question, report a bug, or request a feature!" href="https://github.com/bytedeck/bytedeck/discussions"><i class="fa fa-fw fa-comments-o"></i>&nbsp;ByteDeck Forum</a></li>
-        <li><a href="https://github.com/bytedeck/bytedeck/blob/develop/CHANGELOG.md"><i class="fa fa-fw fa-history"></i>&nbsp;Changelog</a></li>
\ No newline at end of file
+        <li><a title="Ask a question, report a bug, or request a feature!" href="https://github.com/bytedeck/bytedeck/discussions"><i class="fa fa-fw fa-comments-o"></i>&nbsp;Discussion Forum</a></li>
+        <li><a href="https://github.com/bytedeck/bytedeck/blob/develop/CHANGELOG.md"><i class="fa fa-fw fa-history"></i>&nbsp;Changelog</a></li>
```

Comment:
Why are you reverting Bytedeck Forum back to Discussion Forum?  Did you gitpull upstream?  Is this branch based from the most recent develop branch? I'm confused what's happening here.

Reply:
I messed up my rebase. So im guessing git pull upstream will default to my messed up rebase

Reply:
fixed